packages/wonder-blocks-pill/src/components/pill.tsx

```diff
@@ -223,56 +228,57 @@ const _generateColorStyles = (clickable: boolean, kind: PillKind) => {
     }
 
     let backgroundColor;
+    let textColor;
 
     switch (kind) {
         case "accent":
-            backgroundColor = semanticColor.status.notice.foreground;
+            backgroundColor = semanticColor.core.background.instructive.default;
```

Comment:
One thing to note is we won't need to create specific themed component tokens for pills, using semantic colors is enough (especially since we'll be moving away from pills)